lldb/tools/yaml2macho-core/LCNoteWriter.cpp

```diff
@@ -0,0 +1,55 @@
+//===-- LCNoteWriter.cpp --------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "lldb/Utility/UUID.h"
+
+#include "LCNoteWriter.h"
+#include "Utility.h"
+
+#include <ctype.h>
+#include <stdlib.h>
+
+#include "llvm/BinaryFormat/MachO.h"
+
+void create_lc_note_binary_load_cmd(const CoreSpec &spec,
```

Comment:
Surprised this lives in its own file? Why not merge this with utility?

Reply:
It's a tiny one method file today, but I can see me adding additional LC_NOTEs in the future.  So I wanted the "thread writer" file which emits LC_THREAD load commands, the "memory writer" file which emits LC_SEGMENTs and the blocks of memory, and the "LC_NOTE writer" which emits LC_NOTE load commands and bytes of the note command payloads.  It makes sense in my head for these three to be in their own files.